frontend/packages/git-service/src/services/github-service.ts

```diff
@@ -69,15 +69,15 @@ export class GithubService extends BaseService {
 
   getRepoFileList = async (): Promise<RepoFileList> => {
     try {
-      const resp = await this.client.git.getTree({
-        // eslint-disable-next-line @typescript-eslint/camelcase
-        tree_sha: this.metadata.defaultBranch,
+      const resp = await this.client.repos.getContents({
         owner: this.metadata.owner,
         repo: this.metadata.repoName,
+        path: '',
```

Comment:
The API doc says: 
>If you omit :path, you will receive the contents of the repository's root directory.

https://octokit.github.io/rest.js/v18#repos-get-contents
Would it be possible to leave out the path?

Reply:
Just checked. There is a lint error on removing, so not possible.